### PR TITLE
10203 - Add schoolCountry field

### PR DESCRIPTION
**Pull request by dmccurdybah**
## Changes made to existing schema:
- Added a required field `schoolCountry` to the 10203 schema.
- `schoolState` is no longer required

[ZenHub Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/12553)

## Pull Requests to update the schema in related repositories
vets-api and vets-website will be updated with following PRs, but will require this to be merged before I can put those together.

### CODE REVIEW

File: src/schemas/22-10203/schema.js [hunk 1]
```diff
@@ -95,7 +101,7 @@ const schema = {
     'degreeName',
     'schoolName',
     'schoolCity',
-    'schoolState',
+    'schoolCountry',
```
Comment (shawkey-daniel) [inline src/schemas/22-10203/schema.js:104]: Don't we still need schoolState?

Comment (dmccurdybah) [reply src/schemas/22-10203/schema.js:104]: It's required in the UI if the country is USA, but it isn't _required_

Comment (shawkey-daniel) [reply src/schemas/22-10203/schema.js:104]: Doh! Gotcha

**Comment by dmccurdybah** [comment]
@thilton-oddball Thanks!

**Pull request merged**